src/runtime-rs/crates/runtimes/src/shim_mgmt/handlers.rs

```diff
@@ -101,3 +110,43 @@ async fn generic_ip_table_handler(
         _ => Err(anyhow!("IP Tables only takes PUT and GET")),
     }
 }
+
+async fn direct_volume_stats_handler(
+    sandbox: Arc<dyn Sandbox>,
+    req: Request<Body>,
+) -> Result<Response<Body>> {
+    let params = Url::parse(&req.uri().to_string())
+        .unwrap()
+        .query_pairs()
+        .into_owned()
+        .collect::<std::collections::HashMap<String, String>>();
+    let volume_path = params.get(DIRECT_VOLUMN_PATH_KEY).unwrap();
+    let result = sandbox.direct_volume_stats(volume_path).await;
+    match result {
+        Ok(stats) => Ok(Response::new(Body::from(stats))),
+        Err(e) => {
+            let builder = Response::builder().status(StatusCode::INTERNAL_SERVER_ERROR);
+            let resp = builder.body(Body::from(e.to_string())).unwrap();
```

Comment:
Since this function returns a `Result`, can you use `?` here rather than `unwrap()`?